### PR TITLE
performance: useNativeDriver for animations

### PR DESCRIPTION
**Pull request by rooque**

### CODE REVIEW

**Comment by rooque** [comment]
This needs (0.40+) React Native.

File: index.js [hunk 1]
```diff
@@ -143,6 +145,17 @@ export default class Search extends Component {
     }
   };
 
+  focusTextInput = () => {
```
Comment (varungupta85) [inline index.js]: Are you supposed to hook the `focusTextInput` and the `cancelTextInputFocus` with the `TextInput`? The two functions are not called anywhere.

**Comment by rooque** [comment]
I don't know why this last commit appeared here, but I am using it this methods in parent component.

**Comment by varungupta85** [comment]
Right now, both of these functions are not used anywhere. I just checked it again. Could you share how/where are these two functions used in the parent component?

**Comment by rooque** [comment]
See #59

**Pull request closed without merging**